frontend/pages/initiative/[id]/index.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import { humanReadableDateFromStandardString } from '@/lib/time'
+
+const pactaClient = await usePACTA()
+const { fromParams } = useURLParams()
+
+const id = presentOrCheckURL(fromParams('id'))
+const prefix = `initiative/${id}/invitations`
+const [
+  { data: initiative },
+] = await Promise.all([
+  useSimpleAsyncData(`${prefix}.getInitiative`, () => pactaClient.findInitiativeById(id)),
+  useSimpleAsyncData(`${prefix}.getInvitations`, () => pactaClient.listInitiativeInvitations(id)),
+  useSimpleAsyncData(`${prefix}.getRelationships`, () => pactaClient.listInitiativeUserRelationshipsByInitiative(id)),
```

Comment:
Stuff like this confirms my previous grumblings we have the wrong API surface, we shouldn't need three API calls to load a simple page. How about just `GET /initiatives/{id}` returns the initiative, invitations, and relationships, to the extent the caller has access?

Reply:
^^